### PR TITLE
fix(route): 修复快递100快递订单追踪路由

### PR DESCRIPTION
**Pull request by Raven-tu**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

 - [快递 100](https://docs.rsshub.app/other.html#kuai-di-100-kuai-di-ding-dan-zhui-zong)
   - `/kuaidi100/track/:number/:id/:phone?`
```routes
/kuaidi100/track/zhongtongkuaiyun/800002552148
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
 修复快递100无法正确获取快递订单信息。添加额外所需 Cookies 。

### CODE REVIEW

**Comment by Raven-tu** [comment]
修改测试路由地址为具体路由
- `/kuaidi100/track/shunfeng/SF1007896781640/0383`

**Comment by Raven-tu** [comment]
修改commit message

**Comment by Raven-tu** [comment]
> Successfully [generated](https://github.com/DIYgod/RSSHub/actions/runs/3239418276) as following:
> 
> http://localhost:1200/kuaidi100/track/shunfeng/SF1007896781640/0383 - Failed

文档展示订单已过期。

File: lib/routes/kuaidi100/utils.js [hunk 1]
```diff
@@ -25,12 +33,24 @@ async function getCookie(ctx) {
...
                 }
             }
         }
 
         ctx.cache.set(wwwid_key, wwwid, 600);
         ctx.cache.set(csrf_key, csrf, 600);
+        ctx.cache.set(globacsrftoken_key, globacsrftoken, 5);
+        ctx.cache.set(dasddocTitl_key, dasddocTitl, 5);
+        ctx.cache.set(dasddocReferrer_key, dasddocReferrer, 5);
+        ctx.cache.set(dasddocHref_key, dasddocHref, 5);
```
Comment (TonyRL) [inline lib/routes/kuaidi100/utils.js:53]: Do you mean 5 mins? The unit here is second.

Comment (Raven-tu) [reply lib/routes/kuaidi100/utils.js:53]: Thank you for your comment. I'm not sure. 600 may be better for Cookies.

```suggestion
        ctx.cache.set(dasddocHref_key, dasddocHref, 600);
```

Comment (Raven-tu) [reply lib/routes/kuaidi100/utils.js:53]: But I haven't tested this situation.

**Pull request merged**